tests/goldrush_test_demo.sh

```diff
@@ -7,11 +7,11 @@ curl -L --output test_reads.fq https://www.bcgsc.ca/downloads/btl/goldrush/test/
 
 # Run this demo to test your GoldRush installation
 echo "Launching GoldRush"
-goldrush run reads=test_reads G=1e6 t=4 p=goldrush_test  -B
+goldrush run reads=test_reads G=1e6 t=4 p=goldrush_test P=0 -B
 
 l50=$(abyss-fac goldrush_test_golden_path.goldpolish-polished.span2.dist500.tigmint.fa.k40.w250.ntLink-5rounds.polished.fa |awk '{print $3}' |tail -n1)
 
-if [ -e goldrush_test_golden_path.goldpolish-polished.span2.dist500.tigmint.fa.k40.w250.ntLink-5rounds.polished.fa ] && [ ${l50} -eq 1 ]; then
+if [ -e goldrush_test_golden_path.goldpolish-polished.span2.dist500.tigmint.fa.k40.w250.ntLink-5rounds.polished.fa ] && [ ${l50} -eq 2 ]; then
```

Comment:
Interesting that this change makes the demo assembly that much worse? What're your thoughts?

Reply:
I think it's just a matter of what golden path reads ends up getting selected. Could be a length bias or maybe content bias. That's something worth investigating down the line.